hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OmUtils.java

```diff
@@ -497,14 +500,27 @@ public static File createOMDir(String dirPath) {
     return dirFile;
   }
 
-  /**
-   * Returns the DB key name of a deleted key in OM metadata store. The
-   * deleted key name is the <keyName>_<deletionTimestamp>.
-   * @param key Original key name
-   * @param timestamp timestamp of deletion
-   * @return Deleted key name
-   */
-  public static String getDeletedKeyName(String key, long timestamp) {
-    return key + "_" + timestamp;
+  public static RepeatedOmKeyInfo stripGdprMetadata(
```

Comment:
Can you add javadoc for this?

Reply:
Thanks for checking @bharatviswa504 . I have updated PR to address this and also renamed the method appropriately while I was writing the javadoc.